.gitignore

```diff
@@ -1,26 +1,33 @@
+# Rust
 /target
 crates/*/target
 **/*.rs.bk
+/benches/target
+/tools/compile_fail_utils/target
+
+# Cargo Rust
 Cargo.lock
 .cargo/config
 .cargo/config.toml
+
+# Build files IDE
```

Comment:
```suggestion
# IDE files
```

(Nit) I'd also go with a more generic "IDE files", since some are used for configuration as well.